internal/runners/manifest/requirements.go

```diff
@@ -0,0 +1,90 @@
+package manifest
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/model"
+	platformModel "github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+)
+
+type requirement struct {
+	Name            string                      `json:"name"`
+	Namespace       string                      `json:"namespace,omitempty"`
+	Version         resolvedVersion             `json:"version"`
```

Comment:
Can we rename this to ResolvedVersion? While reviewing I kept stumbling over this thinking we were printing the version only rather than requested versus actual. Obviously you're handling both, just the name is missleading which makes this harder to grok.